Edited new AWS container doc for global consistency

FlowWriter opened the pull request:
Reviewed per Pierre's request to create consistency: 

* numbered the steps
* put borders around images
* defined images directory at beginning of document (as opposed to prefixing it to each image file)  

Changes to Kaui doc was only to add a link to the Kaui document outline

Commit:
Update Kaui_Guide_Draft (4) (1).adoc

Commit:
add instructions for making invoice payment

Commit:
edited for consistency across docs

Commit:
editing for consistency across docs

Commit:
Update aws-container.adoc

Commit:
Update aws-container.adoc

ghost commented:
NOTE: The image files will not display the borders until the images are updated on killbill:v3. 

Also, @jdmooney, for the step in which they need to copy the contents of a .json file, it's not clear to me where the contents should be copied. Can you clarify?

jdmooney commented:
I have looked through the serverless option document. I am not at all familiar with this option or with ECS. I tried to follow the instructions in the document, but I cannot find the right Marketplace page, if there is one, and there is no link. Without this I could not create a cluster.
I assume that selecting the JSON option would open a box into which the given file can be pasted. But I cannot verify this.

pierre commented on userguide/tutorials/aws-container.adoc:
```diff
@@ -8,48 +15,68 @@ While we recommend using our https://docs.killbill.io/latest/aws-cf.html[Cloudfo
+image:ecs-cluster-name.png[align=center]
 
-image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ecs-cluster-name.png[align=center]
+If these above steps are succesfful, AWS displays a success screen:
```
`ff`

Commit:
Update ecs-cluster.png

Commit:
Update Kill-Bill-Glossary.adoc

Commit:
Update Kill-Bill-Glossary.adoc

Commit:
standardize single-tier and multi-tier

did a global search/replace (except for where it's used in a filenname/reference)

Commit:
Revert "standardize single-tier and multi-tier"

This reverts commit dc8f694624ff7772f60c741c5559a80d33fc1392.

Commit:
Revert "Update Kill-Bill-Glossary.adoc"

This reverts commit 697708245530d4736aafa8739c82229837abee53.

Commit:
Revert "Update Kill-Bill-Glossary.adoc"

This reverts commit 3ea9c329c5bbfd70f1617990bce72098ceb6ea17.

Commit:
Revert "Update Kaui_Guide_Draft (4) (1).adoc"

This reverts commit 59888efbc15b352f7367964cb744859ba192d0d3.

Commit:
Revert commit for Kaui Guide changes

revert

Commit:
Revert commit for Kaui Guide changes

revert

Commit:
Update Kaui_Guide_Draft (4) (1).adoc

ghost commented:
I'm closing this because it has other commits tangled up in it.
